queenbee/cli/repository/manage.py

```diff
@@ -0,0 +1,70 @@
+import os
+import json
+from urllib.parse import urlparse
+
+try:
+    import click
+    from click_plugins import with_plugins
+except ImportError:
+    raise ImportError(
+        'click modules not installed. Try `pip install queenbee[cli]` command.'
+    )
+
+from ...config.repositories import RepositoryReference
+from ...repository.index import RepositoryIndex, RepositoryMetadata
+
+@click.command('add')
+@click.argument('name')
+@click.argument('path')
+@click.option('-f', '--force', help='Boolean toggle to overwrite existing repository with same name', default=False, type=bool, is_flag=True)
+def add(name, path, force):
+    """add a package repository
+
+    Use this command to add a repository to your local index. This
+    is useful to search, develop and use queenbee packages locally.
+    """
```

Comment:
I could really use some `Args:` here for `name` and `path` that will show up in the click `--help`. As long as you use `\n` for line breaks [like you see here](https://github.com/ladybug-tools/honeybee-energy/blob/master/honeybee_energy/cli/simulate.py#L58-L63) it show up nicely in both the click docs and the sphinx docs.

Reply:
Oh interesting! I've not been documenting my `args` at all throughout the queenbee cli. I have raised an issue to track is separately here #142